packages/flutter/test/services/autofill_test.dart

```diff
@@ -148,6 +148,11 @@ class FakeAutofillClient implements TextInputClient, AutofillClient {
     latestMethodCall = 'performAction';
   }
 
+  @override
+  void performPrivateCommand(String action, Map<String, dynamic> data) {
+    latestMethodCall = 'performPrivateCommand';
```

Comment:
FYI @LongCatIsLooong, small autofill change.

Reply:
Not sure I understand what this is for. Is this for autocomplete text fields?

Reply:
Ah nvm I see it's for https://developer.android.com/reference/android/view/inputmethod/InputConnection?hl=uk#performPrivateCommand(java.lang.String,%20android.os.Bundle)

Reply:
This test probably belongs to "text_input_test.dart" file (in the same directory).

Reply:
Adding the performPrivateCommand() in here because we introduced an abstract function performPrivateCommand in TextInputClient. This is used for resolving the compilation error.